README.md

```diff
@@ -1,6 +1,101 @@
-openmrs-module-openconceptlab
-=============================
+## OpenMRS Open Concept Lab module
 
-See project page at https://wiki.openmrs.org/x/XYA5B
+### Project page
 
-The current development board is at https://issues.openmrs.org/secure/RapidBoard.jspa?rapidView=93
+[OCL Subscription Module (Design Page)](https://wiki.openmrs.org/pages/viewpage.action?pageId=70877277)
+
+### Development board
+
+[Development board](https://issues.openmrs.org/secure/RapidBoard.jspa?rapidView=93)
+
+### Setup Development Environment for Open Concept Lab Module
+
+#### 1.Dependencies:
+- MySQL
+- Maven
+- JDK 1.8
+
+#### 2.Setup OpenMRS server
+
+Run this command to start OpenMRS server setup:
+````sh
+$ mvn openmrs-sdk:setup -DserverId=refapp -Dversion=2.3
+# Note: Pick default values for everything except MySQL username and password
+````
+
+If there are any issues with server setup, check out <b>[OpenMRS SDK documentation](https://wiki.openmrs.org/display/docs/OpenMRS+SDK)</b>
+
+#### 3.Installing Reference Application on server
+
+a) Clone <b>[Reference Application project](https://github.com/openmrs/openmrs-distro-referenceapplication)</b> from GitHub:
+````sh
+$ git clone https://github.com/openmrs/openmrs-distro-referenceapplication.git
+````
+b) Inside Reference Application project directory:
+````sh
+$ mvn clean install -DskipTests
+````
+c) Run server:
+````sh
+$ mvn openmrs-sdk:run
+# Note: Don't visit localhost:8080/openmrs yet!
+````
+d) Build *.omod packages:
+````sh
+mvn clean package -DskipTests
+````
+e) Copy all `*.omod` packages from `/package/target/distro` to local OpenMRS module repository (default `home/.OpenMRS/modules`)
```

Comment:
Do "mvn clean install openmrs-sdk:install -DserverId=refapp" instead of d) and e)